libs/shared/metrics/glean/src/registry/subplat-backend-metrics.yaml

```diff
@@ -3,6 +3,21 @@
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
```

Comment:
Not sure if subplat-backend should also be updated, but I went ahead and added for consistency.

Reply:
@StaberindeZA